hangman.js

```diff
@@ -1 +1,21 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("Welcome to Robizzy's HangMan console game.");
+
+var word = prompt("Please enter a word");
+var attempt = [];
+var incorrectAttemptLimit = 7;
+var alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h',
+        'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's',
+        't', 'u', 'v', 'w', 'x', 'y', 'z'];
+
+function attempt(letter){
```

Comment:
think there might be an issue with name collision here as you have defined a both a variable `attempt` and a function `attempt`, maybe change one of them to `guess`?